cedar-policy/src/tests.rs

```diff
@@ -17,6 +17,7 @@
 #![cfg(test)]
 // PANIC SAFETY unit tests
 #![allow(clippy::panic)]
+#![allow(clippy::cognitive_complexity, clippy::too_many_lines)]
```

Comment:
yeah, these lints are definitely bogus in test code.

When I run clippy,  I never see lints in `#![cfg(test)]` code (though running with `--fix` does fix lints in it). Is there something different about how you're running it?

Reply:
I always enable `--fix`.